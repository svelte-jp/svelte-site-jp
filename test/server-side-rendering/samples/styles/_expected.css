div.svelte-bzh57p{color:red}