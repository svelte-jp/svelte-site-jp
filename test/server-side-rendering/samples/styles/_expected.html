<div class="svelte-bzh57p">red</div>